apps/web/app/(dashboard)/(workspace)/workspace/[workspaceId]/new-sentiment/layout.tsx

```diff
@@ -1,22 +1,37 @@
 "use client";
 
-import { type ReactNode } from "react";
+import { use, type ReactNode } from "react";
 import { Tabs, TabsList, TabsTrigger } from "@repo/ui/components/tabs";
 import { usePathname, useRouter } from "next/navigation";
 import Header from "@repo/ui/components/header";
+import { WorkspaceIdParams } from "@/types/pageParams";
 
-const tabs = [
-  { tab: "manual", url: "/new-sentiment/manual", label: "Manual Entry" },
-  {
-    tab: "file-upload",
-    url: "/new-sentiment/file-upload",
-    label: "File Upload",
-  },
-];
+function getTabs(workspaceId: string) {
+  return [
+    {
+      tab: "manual",
+      url: `/workspace/${workspaceId}/new-sentiment/manual`,
+      label: "Manual Entry",
+    },
+    {
+      tab: "file-upload",
+      url: `/workspace/${workspaceId}/new-sentiment/file-upload`,
+      label: "File Upload",
+    },
+  ];
```

Comment:
minor: we can narrow the type of the constants by applying `as const` statement

```suggestion
  ] as const;
```